app/assets/stylesheets/messages.scss

```diff
@@ -0,0 +1,26 @@
+// Place all the styles related to the Messages controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.incoming-messages {
+  margin: 10px 0;
+  height:60px;
+  color: red;
+  border: 1px red solid;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.